Lib/test/test_logging.py

```diff
@@ -3901,6 +3901,19 @@ def test_handlers(self):
         self.assertIs(handlers[2].formatter, f)
         self.assertIs(handlers[0].formatter, handlers[1].formatter)
 
+    def test_force(self):
+        with support.captured_stdout() as output:
+            logging.basicConfig(stream=sys.stdout)
+            logging.warning('warn')
+            logging.info('info')
```

Comment:
Suggestion: Can we log on `debug` on this line? This way is very clear that the `INFO:root:info` comes after calling `logging.basicConfig(level=logging.INFO, stream=sys.stdout, force=True)` and not before.

Reply:
Good point. Perhaps better yet, we could assert that `output.getvalue()` contains only the `WARNING` line after this call, or immediately before the `logging.info()` call which follows this one.